future-apps/payroll/contracts/Payroll.sol

```diff
@@ -1,36 +1,48 @@
-pragma solidity 0.4.18;
+pragma solidity 0.4.24;
 
 import "@aragon/os/contracts/apps/AragonApp.sol";
-import "@aragon/os/contracts/common/Initializable.sol";
-import "@aragon/os/contracts/common/IForwarder.sol";
+// import "@aragon/os/contracts/common/IForwarder.sol";
```

Comment:
I understand the need to have this workaround now for coverage but make sure to note and re-implement this ASAP.